save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -405,6 +428,8 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         state.bytesReceived = state.availableFiles.sumOf { it.sizeBytes }
         state.isUploading = false
         state.isEditDisabled = true
+        state.selectedMenu = ProjectMenuBar.RUN
+        state.isOpenMenuStatistic = true
```

Comment:
This looks strange. Why are you initializing opened menu to `RUN`, but `isOpenMenuStatistic` implies that statistic is open too?